RecoBTag/SecondaryVertex/python/candidateBoostedDoubleSecondaryVertexCA15Computer_cfi.py

```diff
@@ -6,9 +6,9 @@
     trackSelectionBlock,
     beta = cms.double(1.0),
     R0 = cms.double(1.5),
-    maxSVDeltaRToJet = cms.double(1.3),
+    maxSVDeltaRToJet = cms.double(1.0),
```

Comment:
this change does not look strongly related to the training version.
It would be worth to mention it in the PR description, if so.